### PR TITLE
Adding Filter Model

### PR DESCRIPTION
**Pull request by jostmey**

### CODE REVIEW

**Comment by jostmey** [comment]
Filter Model achieves 60% accuracy on a validation set that I split from the training set. I have not run the model on the test set.
